pages/linux/dm-tool.md

```diff
@@ -0,0 +1,20 @@
+# dm-tool
+
+> A tool to commnicate with the display manager. EG: lock currect session and show lightdm, list all open sessions, connect an existing x server to the display manager.
+> More information: <https://manned.org/dm-tool.1>.
+
+- Show the greeter while keeping current desktop session in ram and waiting to be restored upon autentication by logged in user:
+
+`dm-tool switch-to-greeter`
+
+- Lock the current session:
+
+`dm-tool lock`
+
+- Swicth user, showing a authentication prompt if required:
+
+`dm-tool switch-to-user {{username}} [{{session}}]`
```

Comment:
but don't we still need to have the `{{session}}` thing as the command doesn't work without it

Reply:
If the man page has `[session]` with square brackets is because it's optional, if it's required:
```suggestion
`dm-tool switch-to-user {{username}} {{session}}`
```